final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import json
+import logging
+from urllib import request, error
+from parser import xml_parser
+
+
+def go_for_rss(url: str) -> 'response':
+    """getting data"""
+    try:
+        response = request.urlopen(url)
+    except error.HTTPError as ex:
+        logging.info(ex)
+        raise
+    return response
+
+
+def output_format(news_articles, json_output=False):
```

Comment:
Please be consistent with typehints. It is better to either typehint everywhere or nowhere

Reply:
removed it for now